tests/unit/ai/review/test_orchestrator.py

```
@@ -220,6 +225,115 @@ def test_run_review_depth2_calls_provider_twice() -> None:
+        changed_files=[
+            ChangedFile(
+                path=f"src/file{index}.py",
+                status="modified",
+                additions=1,
+                deletions=0,
+            )
+            for index in range(4)
+        ],
+        unified_diff="diff",
+        pr_metadata=None,
+        repo_root="/tmp/repo",
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/repo"

(S108)